R/classTree.R

```diff
@@ -21,7 +21,7 @@
 #' @return Tree
 #'
 
-BuildTree <-
+classTree <-
```

Comment:
Do we need a rename here?

Reply:
it was easier this way to develop at the time. Random Forest in CRAN and Breiman's original implementation have classTree and regTree as classes for building classification and regression trees. Now, the two files share a lot of similarities so it would make much more sense to have base tree building class but I didn't have time to do that. Otherwise we risk muddying up a single function with a bunch of switch statements and debugging/developing on that is a nightmare.